Tests/Songs/Controller_SongController_UpdateSong.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Controller.DbControllers;
+using Model.Database.Contexts;
+using Model.DbModels;
+using NUnit.Framework;
+using Assert = NUnit.Framework.Assert;
+
+namespace Tests.Playlists
+{
+    [TestFixture]
+    public class Controller_SongController_UpdateSong
+    {
+
```

Comment:
waarom is dit leeg?

Reply:
oeps, die had ik nog ff weg moeten halen. dat roept namelijk gewoon updateItem aan en die wordt al getest

Reply:
oke toppie